data/presets_default.json

```diff
@@ -280,7 +210,13 @@
       "logic_rusted_switches",
       "logic_windmill_poh",
       "logic_crater_bean_poh_with_hovers",
-      "logic_forest_vines"
+      "logic_forest_vines",
```

Comment:
This appears to be a change to Scrub tournament settings.

Reply:
(Let me know if I should undo these changes or push as-is.)

Reply:
As discussed in Discord, these were intentional. If the preset needs to be changed in the future, it'll be in a separate PR.